backend/src/LibExecutionStdLib/LibJson.fs

```diff
@@ -13,4 +9,38 @@ let fn = FQFnName.stdlibFnName
 
 let incorrectArgs = Errors.incorrectArgs
 
-let fns : List<BuiltInFn> = []
+let varA = TVariable "a"
+
+
+let fns : List<BuiltInFn> =
+  [ { name = fn "Json" "serialize" 0
+      typeArgs = [ TVariable "a" ]
```

Comment:
I feel like there's a use to this being a `List<DType>`, but can't put my finger on why, which makes me think typeArgs here could simply be an `int` of the expected count.

Reply:
Thinking out loud:

- functions may take 0 or more type params
- the implementation of a stdlib function will expect that exact # of arguments, and won't worth if another is provided
- it's useful to have this # defined per function, so that the type checker can complain if another # of type args was provided. (currently handled in the interpreter via the "ensure we have the expected # of typeArguments _and_ arguments values" error message)
- right now, I'm modeling this via a `List<DType>`
- but that doesn't really make sense - if anything, a list here would be a list of _constraints_, if/when we support that eventually
  e.g. `let min<'T when 'T : comparison> (x:'T) (y:'T) = if x < y then x else y
- since we don't support type constraints right now, all fn type (definition) parameters are basically "any" constraints
- long term, I could see this being `typeArgs: List<TypeConstraint>`, with `type TypeConstraint = None | Comparable | ...`
- for now, we can just model we could model as an `int` capturing the expected type arg count, _or_ capture this with a tiny `List<TypeConstraint`, with `type TypeConstraint = | NoConstraint`

Reply:
added another commit which might be overkill, but at least makes more sense.